### DISCUSSION TITLE
Wayland/X11 - Diferences fastfetch vs system information

### DISCUSSION THREAD

**Discussion by Walduss** [fastfetch-cli/fastfetch#1073] [General]
Hi all, 
I am quite new in linux, so i dont know if this a bug or I am not understanding properly.

 I am using Wayland when I login my Fedora Linux, but in Fastfetch i see VM=Mutter(X11).  Cheking in Setting/System informtion is shown Wayland.
 
 is there any bug in fastfetch?
 
 thanks in advance .

**Comment by CarterLi** [comment]
It's a bug.

Please file a new issue, and fill the issue template carefully. Thanks.
